lib/file_validator.rb

```diff
@@ -78,9 +78,10 @@ def expected_files(scope)
       expected_files << "files.tgz" if has_files_tarball
       expected_files += tree_tarballs
     else
-      expected_files = files.
-        reject { |file| file["changes"].include?("deleted") }.
-        map { |file| file["name"] }
+      expected_files = files.reject do |file|
+          file["changes"].include?("deleted") ||
+            (file["type"] && file["type"] != "file")
+        end.map { |file| file["name"] }
```

Comment:
`end` at 84, 8 is not aligned with `files.reject do |file|` at 81, 23 or `expected_files = files.reject do |file|` at 81, 6<br>Avoid multi-line chains of blocks.